centaur/src/test/scala/centaur/s3filechecking/NegTestAwsFileChecker.scala

```diff
@@ -0,0 +1,46 @@
+package centaur.s3filechecking
+
+import java.util
+
+import centaur.test.AwsS3Ops
+import com.amazonaws.services.s3.AmazonS3
+import com.amazonaws.services.s3.model.{ObjectListing, S3ObjectSummary}
+import org.scalamock.scalatest.MockFactory
+import org.scalatest.FlatSpec
+import org.specs2.matcher.ShouldMatchers
+
+class NegTestAwsFileChecker extends FlatSpec with ShouldMatchers with MockFactory {
```

Comment:
Please use `spec2.Mokito` since it is already defined as a dependency.
```scala
import org.specs2.mock.Mockito
import org.mockito.Mockito._

class NegTestAwsFileChecker extends FlatSpec with ShouldMatchers with Mockito { /**/ }
```